package.json

```diff
@@ -75,12 +79,12 @@
             "prettier": "^1.14.3"
         },
         "blocks": [
-            ".eslintignore",
             ".eslintrc.js",
-            ".lintstagedrc.js",
+            ".eslintignore",
             ".huskyrc.js",
-            ".prettierrc.js",
             "jest.config.js",
+            ".prettierrc.js",
+            ".lintstagedrc.js",
```

Comment:
Just wondering if these were manually changed? The base config manages these

Reply:
I don't remember changing these as I know they are managed elsewhere. I've changed them back to the order they were originally.